### ISSUE TITLE
Incorrect date-time offset on tooltips - DateAxis tooltip

### ISSUE THREAD

**Issue by ramoakoatta** [amcharts/amcharts4#2919]
**Question**
How do I override the DateAxis tooltip to just show the actual JSON datetime data points?

As an example I have the following JSON data points:

0: {dateTimeBin: Wed Aug 19 2020 06:00:00 GMT+0800 (Australian Western Standard Time), Value: 30390}
1: {dateTimeBin: Wed Aug 19 2020 18:00:00 GMT+0800 (Australian Western Standard Time), Value: 43890}
2: {dateTimeBin: Thu Aug 20 2020 06:00:00 GMT+0800 (Australian Western Standard Time), Value: 44420}
3: {dateTimeBin: Thu Aug 20 2020 18:00:00 GMT+0800 (Australian Western Standard Time), Value: 45040}

When I hover over the column charts, from the first to the last I see the following data points in the tooltip

19 Aug 2020, 12:00,
20 Aug 2020, 00:00,
20 Aug 2020, 12:00,
21 Aug 2020, 00:00,

However, I expect the following:
19 Aug 2020, 06:00,
19 Aug 2020, 18:00,
20 Aug 2020, 06:00,
20 Aug 2020, 18:00,

Basically I expect the tooltip values to just match the dateTimeBin data points. Is there anyway I can override the default tooltip values to just show my expected values matching the JSON points either via an adapter or tooltipLocation/template location setting? Or do I need to explicitly specify a baseInterval setting for this? I've read articles, links below, on moving the tooltip location but that doesn't seem to change the value of the tooltip?

https://github.com/amcharts/amcharts4/issues/1415
https://github.com/amcharts/amcharts4/issues/2210

Cheers

**Environment (if applicable)**

**Additional context**
am4charts.XYChart
am4charts.ColumnSeries

**Comment by martynasma** [comment]
Can you provide a CodePen/jsFiddle of your actual chart? That data does not look like a real JSON.

**Comment by ramoakoatta** [comment]
.

> Can you provide a CodePen/jsFiddle of your actual chart? That data does not look like a real JSON.

@martynasma I'll see if I can replicate the issue on CodePen/jsFiddle. Just to be clear, the data points I posted earlier were just the interpolated output from the browser console. A sample of the actual JSON is below, cheers:

`[
{
	"dateTimeBin": "2020-08-19T06:00:00",
	"value": 30390
},
{
	"dateTimeBin": "2020-08-19T18:00:00",
	"value": 43890
},
{
	"dateTimeBin": "2020-08-20T06:00:00",
	"value": 44420
},
{
	"dateTimeBin": "2020-08-20T18:00:00",
	"value": 45040
}]`

**Comment by martynasma** [comment]
Just to make sure, you are talking about column tooltip or chart cursor tooltip on DateAxis?

**Comment by ramoakoatta** [comment]
> Just to make sure, you are talking about column tooltip or chart cursor tooltip on DateAxis?

It's a column tooltip. Example is the black-background colored tooltip along the base of the x-axis in the link below:
https://www.amcharts.com/docs/v4/tutorials/show-column-tooltip-on-category-hover/

In my case, the tooltip shows the date-time values along the base of the x-axis.

**Comment by martynasma** [comment]
Thanks. Those are axis tooltips.

DateAxis does not show grid/labels at where data is but rather in natural increments based on a number of criteria, like for instance available space, data granularity, etc.

More information is here:
https://www.amcharts.com/docs/v4/concepts/axes/date-axis/#Data_granularity

I would start by setting granularity of your data to 1 hour, as well as increasing grid density.

```JavaScript
dateAxis.baseInterval = { timeUnit: "hour", count: 1 };
dateAxis.renderer.minGridDistance = 30;
```

**Comment by ramoakoatta** [comment]
> Thanks. Those are axis tooltips.
> 
> DateAxis does not show grid/labels at where data is but rather in natural increments based on a number of criteria, like for instance available space, data granularity, etc.
> 
> More information is here:
> https://www.amcharts.com/docs/v4/concepts/axes/date-axis/#Data_granularity
> 
> I would start by setting granularity of your data to 1 hour, as well as increasing grid density.
> 
> ```js
> dateAxis.baseInterval = { timeUnit: "hour", count: 1 };
> dateAxis.renderer.minGridDistance = 30;
> ```

Right, I'll try those settings. Thanks.
